feat: Add actions for checking title and description (#3631)
kyma-project/busola: docs/contributor/testing-strategy.md

- `PR Unit Tests / run-unit-test` - Performs unit tests of the Busola.
- `Lint Markdown Links PR / markdown-link-check` - Checks links in documentation.
- `CodeQL / Analyze (javascript)` - Code quality static code check.

> - `PR Title Check / check-pr-type` - Checking the appropriate prefix for a PR title.

Reviewer: ```suggestion
- `PR Title Check / check-pr-type` - Checks the PR title prefix.
```

> - `Check linked issues / check-linked-issue` - Checking if pull request is linked to issue.

Reviewer: ```suggestion
- `Check linked issues / check-linked-issue` - Checks if PR is linked to the issue.
```

> - `Definition of done Check / check-unchecked-tasks` - Checking if all definitions of done boxes are checked.

Reviewer: ```suggestion
- `Definition of done Check / check-unchecked-tasks` - Checks if all definitions of done boxes are checked.
```

> - `Description Check / check-description` - Checking if the description is not empty.

Reviewer: ```suggestion
- `Description Check / check-description` - Checks if the description is not empty.
```
